aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SdkConfigDecorator.kt

```diff
@@ -54,6 +54,7 @@ class SdkConfigDecorator : RustCodegenDecorator<ClientProtocolGenerator, ClientC
                         builder.set_sleep_impl(input.sleep_impl());
                         builder.set_credentials_provider(input.credentials_provider().cloned());
                         builder.set_app_name(input.app_name().cloned());
+                        builder.set_http_connector(input.http_connector().cloned());
```

Comment:
This is creating a dependency between these two decorators, although it looks like that's already happened with app name, sleep impl, etc. Ideally, the decorators would add their own fields to this via another customization, but I understand if you want to punt on this.

Reply:
I'm doing this as part of https://github.com/awslabs/smithy-rs/pull/1933
I'll update whatever PR gets merged last.